src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorStatementOfTruthPaySubmit.java

```diff
@@ -81,6 +87,15 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             details.getId()
         );
 
+        log.info("Setting dummy payment to mock payment process");
+        if (caseData.getPayments() == null || caseData.getPayments().isEmpty()) {
+            List<ListValue<Payment>> payments = new ArrayList<>();
+            payments.add(solicitorSubmitPetitionService.getDummyPayment(orderSummary));
+            caseData.setPayments(payments);
+        } else {
+            caseData.getPayments().add(solicitorSubmitPetitionService.getDummyPayment(orderSummary));
```

Comment:
so if there is a payment in the list we will add another one?

Reply:
Yeah, from what I can gather from other use services use cases, if a payment were to fail for one reason or another we would want to store it in our payments array for future records - so any subsequent payments would be added to the payments array and we have a record of attempted and successful payments on the case.